com.ibm.streamsx.inet/com.ibm.streamsx.inet.ftp/FTPCommand/FTPCommand_cpp.cgt

```diff
@@ -96,7 +134,7 @@ void MY_OPERATOR::prepareToShutdown() {
 
 // Tuple processing for non mutating ports
 void MY_OPERATOR::process(Tuple const & tuple, uint32_t port) {
-	SPLLOG(L_TRACE, "process(" << tuple << ", " << port << ")", debugAspect);
+	SPLAPPTRC(L_TRACE, "process(" << tuple << ", " << port << ")", debugAspect);
```

Comment:
'Should this be protected with a test to see if we are logging at this level.'
No the the C macro SPLAPPTRC achieves that for us. SPLAPPTRC call makro APPTRC.

'does Streams trace tuples at fine levels anyway?' No at least not in the trace file. Here is a sample with trace level:
23 May 2016 10:33:12.918 [9326] TRACE #splapptrc,J[1],P[11],ResultStream,FTPCommand M[ResultStream.cpp:process:66]  - process({command="mkdir",file="archive",file2=""}, 0)
23 May 2016 10:33:12.918 [9326] TRACE #splapptrc,J[1],P[11],ResultStream,FTPWrapper,FTPCommand M[FTPWrapper.cpp:perform:366]  - Perform FTP action url="ftp://localhost/ftptry/"
23 May 2016 10:33:12.918 [9326] TRACE #splapptrc,J[1],P[11],ResultStream,FTPWrapper,FTPCommand M[FTPWrapper.cpp:headerCallback:875]  - header has=1 count=40 bytes
23 May 2016 10:33:12.918 [9326] DEBUG #splapptrc,J[1],P[11],ResultStream,FTPWrapper,FTPCommand M[FTPWrapper.cpp:headerCallback:880]  - ftp command returns:"550 Create directory operation failed.\r\n"
23 May 2016 10:33:12.919 [9326] ERROR #splapptrc,J[1],P[11],ResultStream,FTPCommand M[ResultStream.cpp:sendError:151]  - "libcurl perform operation failed! curl says: Quote command returned error url:ftp://localhost/ftptry/ command:mkdir archive "
23 May 2016 10:33:12.919 [9326] TRACE #splapptrc,J[1],P[11],ResultStream,FTPCommand M[ResultStream.cpp:process:66]  - process({command="rename",file="file.txt",file2="archive/file.txt